src/js/flows/search/mod.ts

```diff
@@ -27,6 +27,8 @@ type annotateArgs = {
 }
 
 export const annotateQuery = (query: string, args: annotateArgs) => {
+  // if query already starts with 'from', we do not annotate it further
+  if (/^from\s*\(?/i.test(query)) return query
```

Comment:
A slightly simpler regular expression should do the job.
```suggestion
  if (/^from[\s(]/i.test(query)) return query
```